graphql_compiler/query_pagination/pagination_planning.py

```diff
@@ -26,14 +55,17 @@ class PaginationPlan(NamedTuple):
     vertex_partitions: Tuple[VertexPartition, ...]
 
 
-# A field with N quantiles has a pagination capacity N / <QUANTILE_REQUIREMENT_FACTOR>
-QUANTILE_REQUIREMENT_FACTOR = 5
+def get_pagination_plan(
+    schema_info: QueryPlanningSchemaInfo, query_ast: DocumentNode, number_of_pages: int
+):
+    """Make a PaginationPlan for the given query and number of desired pages if possible.
 
+    Raises GraphQLPaginationError if the statistics object is misconfigured.
```

Comment:
I think the `Raises` section generally comes at the end of the docstring per the rest of the project's convention. We should probably also include an `Args` and `Returns`.

Reply:
I can put Raises at the end. I don't think there's a need to document args